### PR TITLE
added the command to send the termination signal to the container

### PR DESCRIPTION
**Pull request by MarshallAsch**
**Describe the pull request:**

The README was missing the instructions on how to stop the docker container. Now it includes the docker build command and the command to send the stop signal to the container.

---

Checklist:

- [x] Run `npm run lint` and ensure linter passes
- [x] Run `npm run test` and ensure tests pass
- [x] Verify that the changes work as expected on Discord
- [x] Update documentation / not applicable
- [x] Update changelog / not applicable

### CODE REVIEW

**Review by SoorajModi — APPROVED** [review]

**Comment by MarshallAsch** [comment]
Do not merge this until after the other fix gets confirmed to prevent it taking botomir down again

**Pull request merged**
